demo-node/entrypoint.bash

```diff
@@ -14,16 +14,14 @@ read -r -u "${NODE[0]}" DUMMY
 # of Tenzir processes.
 export TENZIR_CONSOLE_VERBOSITY="${TENZIR_CLIENT_CONSOLE_VERBOSITY:-"info"}"
 
-tenzir 'remote version | put version | write json'
-
 # Wait until the HTTP endpoint is listening.
 while ! timeout 100 bash -c "echo > /dev/tcp/127.0.0.1/5160"; do
   echo "Waiting for the HTTP service..."
   sleep 1
 done
 
 suricata_pipe="shell 'bash -c \\\"curl -s -L https://storage.googleapis.com/tenzir-datasets/M57/suricata.tar.zst | tar -x --zstd --to-stdout\\\"' | read suricata | where #schema != \\\"suricata.stats\\\" | import"
-zeek_pipe="shell 'bash -c \\\"curl -s -L https://storage.googleapis.com/tenzir-datasets/M57/zeek.tar.zst | tar -x --zstd; cat Zeek/*.log\\\"' | read zeek-tsv | import"
+zeek_pipe="shell 'bash -c \\\"curl -s -L https://storage.googleapis.com/tenzir-datasets/M57/zeek.tar.zst | tar -x --zstd; sync; cat Zeek/*.log\\\"' | read zeek-tsv | import"
```

Comment:
Can you add a comment about `sync`, as it transcends the average person's knowledge?